Packs/EclecticIQ/pack_metadata.json

```diff
@@ -2,11 +2,11 @@
     "name": "EclecticIQ Platform",
     "description": "Threat Intelligence Platform that connects and interprets intelligence data from open sources, commercial suppliers and industry partnerships.",
     "support": "xsoar",
```

Comment:
for adopting the pack:
change support to "partner"

Reply:
Updated